### PR TITLE
Add incentives for Xcel EV rebates

### PR DESCRIPTION
**Pull request by cjohnsbrown**
## Description
[Asana task](https://app.asana.com/0/1207348863918210/1207519219019879/f)

This change adds incentives for the Xcel EV rebates. Eligibility is based on county income threshold, so the json file was updated using data from [here](https://xcelnew.my.salesforce.com/sfc/p/#1U0000011ttV/a/8b000002b0oC/hZ9ON6pdietK9e62ETAHRbT2_kYrddyKhDb7KT6zIQ0) and FIPS codes from [here](https://unicede.air-worldwide.com/unicede/unicede_colorado_fips_3.html)
## Test Plan
* `yarn test`
* validated new incentives show up using local embed

### CODE REVIEW

File: data/CO/incentives.json [hunk 1]
```diff
@@ -12927,5 +12927,53 @@
...
+    "amount": {
+      "type": "dollar_amount",
+      "number": 3000
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Income-qualified customers can receive $3,000off a new EV when you buy or lease from a Colorado-based car dealer."
```
Comment (veekas) [inline data/CO/incentives.json:12975]: typo here

File: data/low_income_thresholds.json [hunk 2]
```diff
@@ -185,6 +185,656 @@
...
+    },
+    "co-xcel-energy-ev": {
+      "type": "county-hhsize",
+      "incentives": [
+        "CO-567",
+        "CO-568"
+      ],
+      "source_url": "https://ev.xcelenergy.com/ev-rebate-co",
+      "thresholds": {
+        "08001": {
```
Comment (veekas) [inline data/low_income_thresholds.json:197]: I need help understanding how this works, and I don't think we have this pattern elsewhere with `thresholds`... can we do a quick huddle in Slack on it tomorrow?

Comment (cjohnsbrown) [reply data/low_income_thresholds.json:197]: Sure we can talk about it. There's some other examples if you do cmd-f in the file for "county-hhsize".

Comment (veekas) [reply data/low_income_thresholds.json:197]: sharing for anyone who sees this later: Cameron showed me the `type` field here that I overlooked, and [this function ](https://github.com/rewiringamerica/api.rewiringamerica.org/blob/main/src/lib/low-income.ts#L14) that checks the type

**Review by veekas — APPROVED** [review]

**Pull request merged**